


package msg;
import "Login.proto";
import "Registration.proto";
import "internal.proto";

message ClientMessage {
  enum ClientMessageType { LoginType = 2; RegistrationType = 3; DisconnectType = 4; ConnectType =5; };
  required ClientMessageType msgType = 1;
  optional Login login = 2;
  optional Registration registration = 20;
  optional Disconnect disconnect = 4;
  optional Connect connect = 5;
  }
